commands/access/add.js

```diff
@@ -25,9 +25,15 @@ function* run(context, heroku) {
   }
 
   if (orgFlags.indexOf('org-access-controls') !== -1) {
-    output += ` with ${cli.color.green(privileges)} privileges`;
     if (!privileges) error.exit(1, `Missing argument: privileges`);
 
+    privileges = privileges.split(",");
+
+    // Give implicit `view` access
+    privileges.push('view');
+    privileges = privileges.sort();
```

Comment:
If I'm reading this correctly, a `push` will end up with a duplicate `view` privilege if a user also includes `view`. You'll have a similar issue in the update command below. To resolve, you can use a `Set`:

```js
let privilegesWithView = new Set(["view", ...privileges.split(",")]);
```

and in the request body:

```js
privileges: [...privilegesWithView]
```

Using a `Set` also provides the advantage of de-duping any privileges a user provides.